utils/translations.js

```diff
@@ -0,0 +1,81 @@
+const fs = require("fs");
+const glob = require("glob");
+const path = require("path");
+
+const SRC_DIR = "./src";
+const LANG_DIR = "./public/lang";
+
+let languages = {};
+
+const searchLanguageFiles = () => {
+  glob.sync(`${SRC_DIR}/**/translations/**/*.json`).map((globFilePath) => {
+    const paths = globFilePath.split("/").reverse();
+
+    const file = paths[0].toLowerCase().replace(".json", "");
+    const lang = paths[1].toLowerCase();
+    const module = paths[3] == "src" ? "__DEFAULT__" : paths[3].toLowerCase();
+
+    const data = fs.readFileSync(globFilePath, "utf8");
+
+    const jsonData = JSON.parse(data);
+    if (jsonData === {}) {
+      return;
+    }
+
+    if (languages[lang] === undefined) {
+      const obj = {};
+      obj[module] = {};
+      obj[module][file] = {};
+      languages[lang] = obj;
+    }
+
+    if (languages[lang][module] == undefined) {
+      languages[lang][module] = {};
+    }
+
+    const obj = {};
+    obj[file] = { ...jsonData };
+
+    languages[lang][module] = {
+      ...languages[lang][module],
+      ...obj,
+    };
+  });
+};
+
+const generateFiles = () => {
+  Object.keys(languages).map((lang) => {
+    const langPath = path.join(LANG_DIR, lang);
+
+    if (!fs.existsSync(langPath)) {
+      fs.mkdirSync(langPath);
```

Comment:
` fs.mkdirSync(langPath, { recursive: true});`

Reply:
Well, it doesn't make that difference 'cause `lang`path will always exists, but... also doesn`t create any negative impact. So, OK, let's do it. :+1: